src/bsp/platform/cc3200/xi_bsp_io_fs_cc3200.c

```diff
@@ -0,0 +1,213 @@
+/* Copyright (c) 2003-2017, LogMeIn, Inc. All rights reserved.
+ *
+ * This is part of the Xively C Client library,
+ * it is licensed under the BSD 3-Clause license.
+ */
+
+#include <xi_bsp_io_fs.h>
+#include <xi_bsp_fwu.h>
+#include <string.h>
+#include <stdio.h>
+
+#include <simplelink.h>
+#include <gpio_if.h>
+
+static _i32 firmware_file_handle_last_opened = 0;
+
+#define XI_DEBUG__FOR_FIRMWARE_UPDATE_TESTING_PURPOSES
```

Comment:
ok, although I don't agree it belongs to header, it is a .c local define which should have effect only on this compilation unit. Headers may be included in other .c / compilation units. I'll move it up.